src/main/host/network/namespace.rs

```diff
@@ -147,7 +147,7 @@ impl NetworkNamespace {
     ) -> Option<impl Deref<Target = NetworkInterface> + '_> {
         if addr.is_loopback() {
             Some(self.localhost.borrow())
-        } else if addr == self.default_ip {
+        } else if addr == self.default_ip || addr.is_unspecified() {
```

Comment:
Maybe worth a comment here that this is a (temporary?) workaround with not-quite-right behavior?

Reply:
Added some comments.